string mapmame 
int64 stamp
---
bool result
